llama-index-packs/llama-index-packs-raft-dataset/llama_index/packs/raft_dataset/base.py

```diff
@@ -0,0 +1,210 @@
+"""RAFT Dataset LlamaPack class."""
+
+# Inspired from https://github.com/ShishirPatil/gorilla/tree/main/raft
+
+from typing import Any
+import random
+from datasets import Dataset
+
+from llama_index.core.llama_pack.base import BaseLlamaPack
+from llama_index.core import SimpleDirectoryReader
+
+# from llama_index.core.node_parser import SemanticSplitterNodeParser
+from llama_index.core.llms import ChatMessage
+from llama_index.llms.openai import OpenAI
+from llama_index.core.node_parser import SimpleNodeParser
+
+DEFAULT_CHUNK_SIZE = 512
+# DEFAULT_BREAKPOINT_PERCENTILE_THRESHOLD = 95 # Use when SemanticSplitterNodeParser
```

Comment:
There is some commented out code, should we remove it?

Reply:
Yeah, I actually intend to use `SemanticSplitterNodeParser` but I think there seems some pydantic issue with it. So for now shifted to simplenodeparser.